src/nfts/seasons/ERC1155SeasonOne.sol

```diff
@@ -67,6 +86,9 @@ contract ERC1155SeasonOne is SeasonsBase {
         // Update total reward tokens needed in contract
         totalRewardTokens -= _rewardAmount;
 
+        // Update total reward tokens used
```

Comment:
No that's not correct. The `verifySolvent` modifier make sure the whole amount of `totalRewardTokens` match the supply of NFT/reward payouts. Without it a user could withdrawn from the contract even if the contract wasnt solvent.

Reply:
Think of it this way. If the contract requires a total of `9_000_000` ztx to be solvent so that all the capsule NFT's can be redeemed, but to redeem just one NFT a user get's `400` ztx. And say the balance of ztx in the contract is `9_000_000 - 1`, without the solvent check it would still be possible to withdraw from the contract as it would have the `400` ztx. The solvent check, checks to make sure the contract was the correct balance to redeem all the supply of capsule NFT's, not just one.